x-pack/test/functional/apps/dashboard/group1/feature_controls/dashboard_security.ts

```diff
@@ -37,7 +37,10 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
 
   describe('dashboard feature controls security', () => {
     before(async () => {
-      await esArchiver.loadIfNeeded('x-pack/test/functional/es_archives/logstash_functional');
+      await esArchiver.loadIfNeeded('x-pack/test/functional/es_archives/logstash_functional', {
+        batchSize: 300,
+        concurrency: 2,
+      });
```

Comment:
Good idea to add this and demonstrate that the change is actually working. Now that we have a green PR build and the reviews are in, I think we can revert this again before the PR is merged, because the performance override is actually not needed here.

Reply:
@wayneseymour will you revert this change? If yes, I suggest we add a an example above function definition so it will be easy to start using it.